KENNEDYMWANGI/API
# #162381042 Set up the database configuration

#### What does this PR do?

 Sets up the database configuration by adding the db_config.py file into the root folder of the application

#### Description of Task to be completed?

Add a db_config file and test successful connection to the database 

#### How should this be manually tested?

Clone the repo and run flask-run - The database should have the tables in the db_config file created successfully.

#### What are the relevant pivotal tracker stories?
#9
